src/sipmpdf/functions.py

```diff
@@ -259,4 +372,17 @@ def sipm_response(
   Using numba for calculation acceleration (the function is difficult to express
   in pure array syntax)
   """
-  pass
+  kern = kernel_switch(x, pedestal, gain, common_noise, pixel_noise, poisson_mean, poisson_borel, ap_prob, ap_beta)
+ 
+  Pdc=0 ##probability of dark current, define here, fix all these values later
+  smear_of_darkcurrent=0 ##define here
+  resolution_of_darkcurrent=0 ##define here
```

Comment:
This should be a new variable that the user will have to define (fixed to 1e-4 by default)